exporter/lokiexporter/config.go

```diff
@@ -84,3 +96,19 @@ func (c *LabelsConfig) getAttributes() map[string]model.LabelName {
 
 	return attributes
 }
+
+// getResourceAttributes creates a lookup of allowed attributes to valid Loki label names.
+func (c *LabelsConfig) getResourceAttributes() map[string]model.LabelName {
```

Comment:
Any change you had thoughts on combining the methods for LogRecordAttributes and ResourceAttributes? If not, no worries at all. Just curious if you had thought through it or not.

Reply:
I could add both loops to a single function and return 2 `map[string]model.LabelName`. We need to distinguish between the map of logRecordAttributes and ResourceAttributes so can't merge the two loop as far as I can tell